script/__snapshots__/verify-changes.test.ts.snap

```diff
@@ -0,0 +1,849 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`verify .all-contributorsrc 1`] = `
+"diff --git a/.all-contributorsrc b/.all-contributorsrc
+index f4d6216..d0fed87 100644
+--- a/.all-contributorsrc
++++ b/.all-contributorsrc
+@@ -1,5 +1,5 @@
+ {
+-  \\"badgeTemplate\\": \\"<a href=\\\\\\"#contributors\\\\\\" target=\\\\\\"_blank\\\\\\"><img alt=\\\\\\"All Contributors: <%= contributors.length %> 🤝\\\\\\" src=\\\\\\"https://img.shields.io/badge/all_contributors-<%= contributors.length %>_🤝-21bb42.svg\\\\\\" /></a>\\",
++  \\"badgeTemplate\\": \\"<a href=\\\\\\"#contributors\\\\\\" target=\\\\\\"_blank\\\\\\"><img alt=\\\\\\"All Contributors: <%= contributors.length %> 👪\\\\\\" src=\\\\\\"https://img.shields.io/badge/all_contributors-<%= contributors.length %>_👪-21bb42.svg\\\\\\" /></a>\\",
+   \\"commit\\": false,
+   \\"commitConvention\\": \\"angular\\",
+   \\"contributors\\": [
+@@ -24,335 +24,253 @@
+       \\"name\\": \\"Jeff Wen\\",
+       \\"avatar_url\\": \\"https://avatars.githubusercontent.com/u/3297859?v=4\\",
+       \\"profile\\": \\"https://sinchang.me\\",
+-      \\"contributions\\": [
+-        \\"code\\"
+-      ]
++      \\"contributions\\": [\\"code\\"]
```

Comment:
[Non-actionable] Hmm, after https://github.com/all-contributors/cli/pull/356 and https://github.com/all-contributors/cli/pull/360 I would have thought all-contributors would be respecting the repo's Prettier config... do you have any idea why there are diffs here? I don't think we need to change anything for this PR around formatting. Just curious.

Reply:
Haven't found the reason for this yet.

Reply:
Me neither 😄 resolving the thread. A mystery for another day.